lib/actions/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2016-2020 Balena
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as apiKey from './api-key';
+import * as app from './app';
+import * as auth from './auth';
+import * as config from './config';
+import * as device from './device';
+import * as help from './help';
+import * as local from './local';
+import * as logs from './logs';
+import * as os from './os';
+import * as push from './push';
+import * as ssh from './ssh';
+import * as tags from './tags';
+import * as tunnel from './tunnel';
+import * as util from './util';
```

Comment:
Ideally these would be `export * as` but that requires balena-lint 5.x which requires node 10+

Reply:
I see!